test/document_stores/test_base.py

```diff
@@ -14,7 +14,7 @@ class DocumentStoreBaseTestAbstract:
     class but only for its subclasses.
     """
 
-    @pytest.fixture
+    @pytest.fixture(scope="class")
```

Comment:
I don't have a proof at hand, but I believe some tests modify the `documents` list. Why setting the scope to `class`? It seems lightweight, I'd rather be safer than faster here